server/src/main/java/io/crate/metadata/GeneratedReference.java

```diff
@@ -63,7 +63,7 @@ public GeneratedReference(StreamInput in) throws IOException {
             ref = new SimpleReference(in);
         }
         formattedGeneratedExpression = in.readString();
-        generatedExpression = Symbols.fromStream(in);
+        generatedExpression = Symbols.nullableFromStream(in);
```

Comment:
This change needs some version checks as this would break streaming compatibility due to the additional null getting serialized.

Reply:
Yep, was thinking that with backports wouldn't need version checks. Pushed a fixup.